src/js/scripts.js

```diff
@@ -1 +1,13 @@
-// Add your scripts here
+const img = document.getElementById('slidewrap');
+const images = img.getElementsByTagName('img');
+let counter = 1;
+if (counter <= images.length) {
+	setInterval(() => {
+		images[0].src = images[counter].src;
+		console.log(images[counter].src);
+		counter += 1;
+		if (counter === images.length) {
+			counter = 1;
+		}
+	}, 1500);
+}
```

Comment:
Nice JavaScript work here! This would also be a great place to use a for loop. That might look something like:

```javascript
for (let i  = 0; i <= images.length; i++) {
    setInterval(() => {
		images[0].src = images[i].src;
		console.log(images[i].src);
	}, 1500);
}
```

A for loop is kind of like a shortcut for setting a variable and creating an if statement like this, except it runs multiple times. Try this - if you're having trouble getting this to run multiple times and set an interval on multiple images, this for loop should help!